cronjobs/cnn_classify_data.py

```diff
@@ -219,7 +233,12 @@ def match_files(cafiles, zbfiles):
     tr_spectrum=None
 
     # Access redux folder.
+
+    for i in range(1):
+        tile_number = tile_numbers
+
```

Comment:
How are these lines helping?

Reply:
These for loop was the main cause of the issue initially (making the code run on individual digits of tilenum instead of taking the whole number as the tilenum), and I made a trivial fix by making it loop only once. SO we could definitely just also remove the for loop, but I wasn't sure if that would mess up with something you had in mind while writing it.

Reply:
I too don't understand what these lines are doing.  A range of 1 is trivial, so why the `for` statement?

This just becomes` tile_number=tile_numbers`.